src/influxdb.c

```diff
@@ -111,6 +112,30 @@ corto_int16 influxdb_serScalar(
     return 0;
 }
 
+corto_int16 influxdb_serComposite(
+    corto_walk_opt *walk,
+    corto_value *info,
+    void *userData)
+{
+    corto_type t = corto_value_typeof(info);
+
+    if (t->kind == CORTO_COMPOSITE) {
+        if (corto_walk_members(walk, info, userData)) {
+            goto error;
+        }
+    } else if (t->kind == CORTO_COLLECTION) {
+        if (corto_walk_elements(walk, info, userData)) {
+            goto error;
+        }
+    } else {
+        corto_warning("Unexpected serialize request for type [%d]", t->kind);
```

Comment:
This would be a critical error. The walk API should _never_ forward a value other than the ones specified in the program and metaprogram.

Suggest to turn this into a `corto_throw`, and `goto error`.